geniza/corpus/management/commands/import_data.py

```diff
@@ -294,8 +307,57 @@ def add_document_language(self, doc, row):
             doc.language_note = "\n".join(notes_list)
             doc.save()
 
+    def import_document(self, row, demerge=False):
+        """Import a single document given a row from a PGP spreadsheet"""
+        if ";" in row.type:
+            logger.warning("skipping PGPID %s (demerge)" % row.pgpid)
+            self.docstats["skipped"] += 1
+            return
+
+        doctype = self.get_doctype(row.type)
+        fragment = self.get_fragment(row)
+        doc = Document.objects.create(
+            id=row.pgpid if row.pgpid else None,
+            doctype=doctype,
+            description=row.description,
+        )
+        doc.tags.add(*[tag.strip() for tag in row.tags.split("#") if tag.strip()])
+        # associate fragment via text block
+        TextBlock.objects.create(
+            document=doc,
+            fragment=fragment,
+            # convert recto/verso value to code
+            side=self.recto_verso_lookup.get(row.recto_verso, ""),
+            region=row.text_block,
+            subfragment=row.multifragment,
+        )
+        self.add_document_language(doc, row)
+        if demerge:
+            self.docstats["demerged_documents"] += 1
```

Comment:
Do we really need a separate count? Since you're incrementing the total for documents, you can save the total before you run import the demerge and then compare the difference after.

Reply:
oh that's smart 💡 thanks!